src/ja/home.html

```diff
@@ -0,0 +1,106 @@
+---
+title: 'Open Web Advocacy'
+metaDesc: 'オープンなウェブの未来を擁護するために世界中から集まったソフトウェアエンジニアのグループ'
+permalink: '/ja/'
+layout: 'layouts/home.njk'
+isHome: true
+---
+
+<p class="home-intro">
+  私たちは<strong>オープンなウェブの未来</strong>を擁護するために世界中から集まったソフトウェアエンジニアのグループです。規制当局、立法者、政策立案者に対して複雑な技術の詳細情報を提供することで、この業界における競争を阻害する主要な問題とその解決方法の理解をサポートしています。
+</p>
+
+<div class="highlight">
+  <h2>危機にさらされているのは何か？</h2>
+
+  <div class="group">
+    <p>アプリケーション開発の未来すべてが危機にさらされています。規制や立法による変化なくしては、普遍的で、無料かつオープンな、一度書くだけでよく、どこにでもデプロイ可能な、アプリケーション頒布とデプロイのシステムを失うリスクがあります。このシステムは、企業と消費者のコストを大幅に削減するシステムです。
+    <p>もし規制や立法による変化がなければ、資金はプロプライエタリなエコシステムに向かい、ゲートキーバーたちは重税を徴収することが可能になります。これにより、ゲートキーバーのコントロール下にロックインされ、今後何年にもわたってモバイルアプリのイノベーションが阻害されることになってしまいます。</p>
```

Comment:
- 文脈的に、"funding"が指してるのは"資金"ではなく"企業"？"ビックテック"に変えてもいいかも
- "ゲートキーパー"の説明を多少入れたほうがいいかも……？
```suggestion
    <p>もし規制や法律の変化がなければ、大企業は独占的な生態系に移行し、「ゲートキーパー」と呼ばれる企業たちは重い"税金"を徴収できるようになります。これにより、ゲートキーバー達の管理化に囚われ、今後長きにわたってモバイルアプリのイノベーションが阻害されることになってしまいます。</p>
```